src/slack_interaction_handler.ts

```diff
@@ -265,12 +260,12 @@ export async function handleVoterStatusUpdate({
         );
       }
 
-      // Replace the entire block so that the initial option change persists.
-      await SlackInteractionApiUtil.replaceSlackMessageBlocks({
-        slackChannelId: payload.channel.id,
-        slackParentMessageTs: payload.container.thread_ts,
-        newBlocks: payload.message.blocks,
-      });
+      // Update the voter block
```

Comment:
What's the thinking behind this comment change?

Reply:
None really--the original comment just got moved into the helper.  Switched it back in the latest commit.